CMakeLists.txt

```diff
@@ -306,14 +306,6 @@ if(GISMO_BUILD_LIB)
   endif()
 endif()
 
-if(GISMO_ELASTICITY)
-  gismo_fetch_module(gsElasticity)
-endif()
-
-if(GISMO_EXASTENCILS)
-  gismo_fetch_module(gsExaStencils)
-endif()
-
```

Comment:
I think there was a reason why this was in this position, does everything still work after this commit ?

Reply:
I have tried to invoke
```
cmake . -DGISMO_ELASTICITY=On
cmake . -DGISMO_EXASTENCILS=On
make
```
in my build folder and it worked (up to the bugs the code had/has anyway).